### PR TITLE
Update Shaman Profile

### PR DESCRIPTION
**Pull request by Billdong1997**

### CODE REVIEW

**Comment by Xian55** [comment]
Hey thanks for the contribution!
Please remove the comments.

File: Shaman_22.json [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+        "Name": "Repair",
+        "Key": "R",
+        "Requirement": "Items Broken",
+        "PathFilename": "NPC_Stonetalon.json",
+        "Cost": 6
+      },
+      {
+        "Name": "Sell",
+        "Key": "T",
+        "Requirement": "BagCount>46",
```
Comment (Xian55) [inline Shaman_22.json:152]: use bag full instead

File: Shaman_22.json [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+        "MinMana": 75,
+        "Requirements": ["not InMeleeRange", "Mana%>30"]
+      },
+	  
+      {
+        "Name": "Healing Wave",
+        "Key": "3",
+        "Requirements": ["Health%<45","TargetHealth%>20"],
+		"Cooldown": 5,
+        "HasCastBar": true,
```
Comment (Xian55) [inline Shaman_22.json:63]: is ususally a good idea to add `"StopBeforeCast": true,` as well

File: Shaman_15.json [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+        "Name": "Repair",
+        "Key": "R",
+        "Requirement": "Items Broken",
+        "PathFilename": "NPC_Cross.json",
+        "Cost": 6
+      },
+      {
+        "Name": "Sell",
+        "Key": "T",
+        "Requirement": "BagCount>46",
```
Comment (Xian55) [inline Shaman_15.json:132]: `BagsFull` instead

File: Shaman_15.json [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+        "Requirements": ["Health%<49","MobCount>2"],
+        "Cooldown": 120,
+        "Log": false,
+        "MinMana": 10
+      },
+      {
+        "Name": "Healing Wave",
+        "Key": "3",
+        "Requirement": "Health%<45",
+		"Cooldown": 5,
```
Comment (Xian55) [inline Shaman_15.json:44]: code indentation
and `"StopBeforeCast": true,`

File: Shaman_22.json (hunk 1 above)
Comment (Billdong1997) [reply Shaman_22.json:152]: Oh yes, you know my "bagfull" command cant work until today. I will revise it now.

**Pull request merged**